src/fadt.rs

```diff
@@ -3,6 +3,8 @@ use sdt;
 use sdt::SdtHeader;
 use {Acpi, AcpiError, AcpiHandler, GenericAddress, PhysicalMapping};
 
+type ExtendedField<T> = sdt::ExtendedField<T, typenum::U1>;
```

Comment:
At the moment, all extended fields in the FADT were added in v2.0, but this isn't very future compatible. Maybe encode the revision in each field?

We could also perhaps define constants like `REVISION_V2` or for the `typenum::U1` somewhere else to make this more readable.

Reply:
> At the moment, all extended fields in the FADT were added in v2.0,

Okay cool, I'll change it to `U2`. 

> this isn't very future compatible

If and when the API gets updated we can change it and write out the revision specifier in each extended field. This is just shorthand

> We could also perhaps define constants like REVISION_V2

We *could*, but is it really worth it? Eh...

Reply:
Fair enough.

I've not tested this because Pebble isn't currently in a state to, but if you've tested this works as is, I'll merge.

Reply:
Let me test this again to be extra sure and then I'll either fix it or we can merge it if it works. :+1:

Reply:
Tested & works.